### PR TITLE
Implement database module #31

### PR DESCRIPTION
**Pull request by HenoJurek**
This is basic implementation that just connect with DB. More advance logic will be implement later - #47

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
+// TODO
```
[the remark this answers is not in the document]
Comment (HenoJurek) [reply controllers/user.js]: this mean all file is to rewrite

File: lib/database.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// TODO
```
[the remark this answers is not in the document]
Comment (HenoJurek) [reply lib/database.js]: this mean all file is to rewrite

File: controllers/user.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Drego31) [reply controllers/user.js]: Write this in TODO comment and that's all ;)

File: lib/database.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Drego31) [reply lib/database.js]: Write this in TODO comment and that's all ;)

[the remark this answers is not in the document]
Comment (HenoJurek) [reply lib/database.js]: ok

File: controllers/user.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (HenoJurek) [reply controllers/user.js]: ok

**Pull request merged**